extensions/mistral/src/utils/mistral-client.ts

```diff
@@ -0,0 +1,6 @@
+import { Mistral } from "@mistralai/mistralai";
+import { getPreferenceValues } from "@raycast/api";
+
+const { apiKey } = getPreferenceValues<{ apiKey: string }>();
```

Comment:
```suggestion
const { apiKey } = getPreferenceValues<ExtensionPreferences>();
```

You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`